### PR TITLE
feat: Enable rc workflow for debian

### PR DESCRIPTION
**Pull request by AsabuHere**
# Fixes #
[DII-539](https://issues.corp.twilio.com/browse/DII-539) Platform exec for rc : deb
A short description of what this PR does.
Enable steps for rc release for deb in twilio-cli
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

**Comment by shrutiburman** [comment]
With Actions, even a syntax error can cause us trouble during release. So, please add a demo run for this new workflow, it makes the review easier. :)

**Comment by AsabuHere** [comment]
RC branch run : https://github.com/AsabuHere/twilio-cli/actions/runs/3966114197

**Review by shrutiburman — APPROVED** [review]

File: .github/workflows/debian-executable-release.yml [hunk 1]
```diff
@@ -19,6 +19,21 @@ jobs:
...
+          echo "::set-output name=version::$(echo "$releaseVersionString" | cut -d- -f1)"
+      - name: Setting file path
+        id: set-file-path
+        run: |
+          if [[ ${{ steps.get-tag.outputs.TAG_NAME }} == *"rc"* ]]; then
+            file_path=dist/deb/${{ github.event.inputs.formula }}_${{ steps.process-release-version.outputs.VERSION }}.${{ steps.get-sha.outputs.SHA_SHORT }}-1_amd64.deb
+          else
+            file_path=dist/deb/${{ github.event.inputs.formula }}_${{ steps.get-tag.outputs.TAG_NAME }}.${{ steps.get-sha.outputs.SHA_SHORT }}-1_amd64.deb
+          fi
+          echo "FILE_PATH=$file_path" >> $GITHUB_OUTPUT
```
Comment (sbansla) [inline .github/workflows/debian-executable-release.yml:36]: Artifact is also uploaded to github, Please do test that as well.

Comment (AsabuHere) [reply .github/workflows/debian-executable-release.yml:36]: Verified upload of github asset to the particular release

**Pull request merged**